### PR TITLE
List service keys operation

### PR DESCRIPTION
**Pull request by LokeshN**
This change adds the operation for listing the service keys for an
instance

### CODE REVIEW

File: cloudfoundry-operations/src/main/lombok/org/cloudfoundry/operations/services/ServiceKey.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+package org.cloudfoundry.operations.services;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * A service key
+ */
+@Data
+public class ServiceKey {
```
Comment (nebhale) [inline cloudfoundry-operations/src/main/lombok/org/cloudfoundry/operations/services/ServiceKey.java]: Class should be `final`.  I'll fix.

File: cloudfoundry-operations/src/test/java/org/cloudfoundry/operations/services/ListServiceKeysRequestTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+package org.cloudfoundry.operations.services;
+
+import org.cloudfoundry.ValidationResult;
+import org.junit.Test;
+
+import static org.cloudfoundry.ValidationResult.Status.INVALID;
+import static org.cloudfoundry.ValidationResult.Status.VALID;
+import static org.junit.Assert.assertEquals;
+
+public class ListServiceKeysRequestTest {
```
Comment (nebhale) [inline cloudfoundry-operations/src/test/java/org/cloudfoundry/operations/services/ListServiceKeysRequestTest.java]: Class should be `final`.  I'll fix.

**Comment by nebhale** [comment]
@LokeshN You've got an impressive gift at choosing problematic stories 😄.  While reviewing this submission, I traced the following requests in the CLI:

```
GET /v2/spaces/<SPACE_ID>/service_instances?return_user_provided_service_instances=true&q=name%3Arabbit-test&inline-relations-depth=1
GET /v2/services/<SERVICE_ID>
GET /v2/service_instances/<SERVICE_INSTANCE_ID>/service_keys
```

Sadly, that final request is a completely undocumented API.  I've got a request into the CAPI team to find out what the deal is here, but until that's sorted out, this issue will be stalled.

**Comment by nebhale** [comment]
Issues opened:

* https://github.com/cloudfoundry/cli/issues/826
* https://github.com/cloudfoundry/cloud_controller_ng/issues/582

**Comment by nebhale** [comment]
@LokeshN Now that the list service instance service keys functionality is in place, why don't you rebase and amend this PR to use it.

**Comment by LokeshN** [comment]
@nebhale Tried to rebase the PR branch but ran into issues, so created a new one.

**Pull request closed without merging**